views/about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- page title -->
+    <title>About</title>
+    <!-- link to ABOUT css file -->
+    <link rel="stylesheet" href="/public/styles/stylesAbout.css" />
+    <!-- Font families   -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <!-- giving an icon to the browser tab -->
+    <link rel="icon" href="/public/images/icon.png" />
+  </head>
+  <body>
+    <header>
+      <div class="navbar-container">
+        <img
+          src="/public/images/tarantino-favicon.png"
```

Comment:
The same as above for every image. You need to remove the `/public` path to correctly display the images on your browser.